src/codewhisperer/service/recommendationHandler.ts

```diff
@@ -399,7 +403,10 @@ export class RecommendationHandler {
                     session.requestIdList,
                     sessionId,
                     page,
-                    editor.document.languageId,
+                    runtimeLanguageContext.getLanguageContext(
+                        editor.document.languageId,
+                        editor.document.fileName.substring(editor.document.fileName.lastIndexOf('.') + 1)
```

Comment:
why not use `path.extname()` ? it is more readable, shorter, and potentially more robust.

Reply:
```suggestion
                        path.extname(editor.document.fileName)
```

Reply:
+1